mslib/msui/mss_pyui.py

```diff
@@ -419,6 +421,11 @@ def closeEvent(self, event):
                 self.mscolab_window.close()
             if self.config_editor is not None:
                 self.config_editor.close()
+            # tv = tableview ; it is an object of MSS Tableview window. Used for force closing it.
+            if self.tv_window is not None:
+                if self.tv_window.tvwindow_exists() is True:  # checks whether or not tableview window has closed before
```

Comment:
below it is written as self.tv_window.tv_window_exists

only exists() would be sufficient.

see for example os.path.exists()

best practise it would be written for booleans if self.tv_window.exists(): see PEP8 too or,
https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_true.html

Reply:
Okay, I am rectifying that exists() part and PEP8 style. But do I need to write the tests?

Reply:
you can try if you want. For bugs it is with starting a test sometimes easier to find the correct fix. So one makes first a test which fails by the bahaviour than the bug fix and the before failing test succeeds. It can also save a lot of time (that time one needs to try out different states). 

It is not needed now. We could later add tests which look on states of different windows when expected closed.

Reply:
Alright then. I will try the tests later when we increase test coverage for other codes. For now, its done. I have already updated the changes that you had requested.